docs/api.rst

```diff
@@ -37,6 +37,7 @@ developers, not a gospel.
     api/pulp_smash.tests.ostree.api_v2.test_crud
     api/pulp_smash.tests.ostree.api_v2.test_publish
     api/pulp_smash.tests.ostree.api_v2.test_sync
+    api/pulp_smash.tests.ostree.api_v2.test_copy
```

Comment:
Nitpick: This list of modules is alpha-numerically sorted. Can this line be moved to the right place?